Modules/System/Base64 Convert/README.md

```diff
@@ -28,6 +28,53 @@ The string representation, in base-64, of the input string.
 
  Converts the value of the input string to its equivalent string representation that is encoded with base-64 digits.
  
+#### Syntax
+```
+procedure ToBase64(String: Text; TextEncoding: TextEncoding): Text
+```
+#### Parameters
+*String ([Text](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/text/text-data-type))* 
+
+The string to convert.
+
+*TextEncoding ([TextEncoding](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/textencoding/textencoding-option))*
+
+The Encoding of the Text. If Windows is selected, ansi (1252) is defaulted unless a codepage is specified.
+
+#### Return Value
+*[Text](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/text/text-data-type)*
+
+The string representation, in base-64, of the input string.
+### ToBase64 (Method) <a name="ToBase64"></a> 
+
+ Converts the value of the input string to its equivalent string representation that is encoded with base-64 digits.
+
+
+#### Syntax
+```
+procedure ToBase64(String: Text; TextEncoding: TextEncoding; Codepage: Integer): Text
+```
+#### Parameters
+*String ([Text](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/text/text-data-type))* 
+
+The string to convert.
+
+*TextEncoding ([TextEncoding](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/textencoding/textencoding-option))*
+
+The Encoding of the Text.
+
+*Codepage ([Integer](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/integer/integer-data-type))*
+
+The codepage if MsDos or Windows are selected as TextEncoding, otherwise ignored.
```

Comment:
MS-DOS

Reply:
But "MsDos" is what the option on TextEncoding is called.